client/lib/plugins/sanitize-section-content.js

```diff
@@ -129,11 +134,8 @@ const replacementFor = node => {
  * @returns {string} sanitized HTML
  */
 export const sanitizeSectionContent = content => {
-	const parser = new DOMParser();
-	const doc = parser.parseFromString( content, 'text/html' );
```

Comment:
this change might be dangerous. DOMParser won't pull and run script tags included in the parsed DOM, while innerHTML probably will... Needs testing.

Try parsing some HTML that includes a script tag and see if it runs.

Reply:
I injected a `script` tag and wasn't able to trigger it on a plugin page, but since I'm not 100% sure it won't make for an unsafe situation, I'll go ahead and pull it out.